SECURITY.md

```diff
@@ -0,0 +1,12 @@
+# Security Policy
+
+## Supported Versions
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 3.x   | :white_check_mark: |
+
+## Reporting a Vulnerability
+
+If you think you've found a potential vulnerability in Elyra or any of its sub-projects, please
+email akchin@us.ibm.com to responsibly disclose it.
```

Comment:
Should elyra-ai have an email or some way to have this be multi-owned?

Reply:
Update to use the security reporting feature in github. Core admins will receive a private issue in GH security tab and decide how to proceed.